pom.xml

```diff
@@ -45,6 +46,11 @@
 
     <dependencies>
         <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
```

Comment:
I believe it was separated out and there was some existing code that depended on it.